SingularityUI/app/actions/api/request.es6

```diff
@@ -0,0 +1,4 @@
+import { buildApiAction, buildJsonApiAction } from './base';
+
+export const FetchAction = buildApiAction('FETCH_REQUEST', (requestId) => {return {url: `/requests/request/${ requestId }`}});
```

Comment:
you can simplify the lambda here like this:

```javascript
(requestId) => ({url: `/requests/request/${ requestId }`})
```

(the parens around the object make the complier do the right thing)